src/site/content/en/blog/gpu/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Access modern GPU features with WebGPU
+subhead: |
+  WebGPU enables high-performance 3D graphics and data-parallel computation on
+  the Web.
+authors:
+  - beaufortfrancois
+  - cwallez
+date: 2021-08-26
+#updated: 2021-08-26
+hero: image/vvhSqZboQoZZN9wBvoXq72wzGAf1/SN6GIsxmcINXJZKszOTr.jpeg
+thumbnail: image/vvhSqZboQoZZN9wBvoXq72wzGAf1/SN6GIsxmcINXJZKszOTr.jpeg
+description: |
+  WebGPU enables high-performance 3D graphics and data-parallel computation on
+  the Web.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+  - games
+feedback:
+  - api
+stack_overflow_tag: webgpu
+---
+
+## What is WebGPU? {: #what }
+
+[WebGPU] is a new API for the web that exposes modern computer graphics APIs
```

Comment:
```suggestion
[WebGPU] is a new web API that exposes modern computer graphics APIs
```

Reply:
Graphics APIs or graphics capabilities? Doesn't WebGPU just wrap lower-level capabilities rather than provide pass-through access?

Reply:
Good call, "computer graphics capabilities" would work well.

Reply:
Done.

Reply:
Both parts?

Reply:
Both the change to 'capabilities' and the text tweek?

Reply:
Yes.